src/Data/Fin/Subset/Properties.agda

```diff
@@ -59,6 +67,51 @@ module _ {n} {p q : Subset n} where
   in⊆in-⇔ : p ⊆ q ⇔ inside ∷ p ⊆ inside ∷ q
   in⊆in-⇔ = equivalence in⊆in drop-∷-⊆
 
+  s⊆s : ∀ {s} →  p ⊆ q → s ∷ p ⊆ s ∷ q
+  s⊆s {.inside} p⊆q here = here
+  s⊆s {s} p⊆q (there ∈p) = there (p⊆q ∈p)
```

Comment:
Don't think you need the implicits here.

Reply:
Done.